lambdas/nino-check/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "nino-check-handler",
+  "description": "",
+  "scripts": {
+    "lint": "eslint .",
+    "lint:fix": "eslint . --fix",
+    "unit": "jest --silent",
+    "test": "npm run unit --",
+    "test:coverage": "npm run unit -- --coverage",
+    "deploy": "../../deploy.sh",
+    "compile": "tsc"
+  },
+  "dependencies": {
+    "@aws-lambda-powertools/logger": "2.3.0",
+    "@aws-lambda-powertools/parameters": "2.21.0",
+    "@aws-sdk/client-dynamodb": "3.828.0",
+    "@aws-sdk/client-eventbridge": "3.828.0",
+    "@aws-sdk/util-dynamodb": "3.828.0"
+  },
+  "devDependencies": {
+    "@types/aws-lambda": "^8.10.150",
```

Comment:
We generally pin all dependency versions - is there a reason you've not for the devDependencies?

Reply:
Nah, none in particular - Mike advised me to pin the deps but didn't seem fussed about dev deps. Happy to pin them now.

Reply:
Had a go at this and had some type errors with Jest. presumably in part because all the other `package.json`s seem to use `^` on them, including the root package.json and common - so if we want to pin dev deps then I'm not sure it's in scope for this ticket. Are you okay with me leaving this for now and maybe we spin up a separate ticket to ensure dev deps are pinned?